GeneralsMD/Code/GameEngine/Source/GameLogic/AI/AIPathfind.cpp

```diff
@@ -7130,6 +7130,11 @@ Path *Pathfinder::findGroundPath( const Coord3D *from,
 	// "closed" list is initially empty
 	m_closedList = NULL;
 
+	// TheSuperHackers @fix helmutbuhler This was originally uninitialized and in the loop below.
+#if defined(_MSC_VER) && _MSC_VER < 1300
```

Comment:
Please use RETAIL_COMPATIBLE_CRC macro here instead. Same at the other place below. This way, this behavior also goes into our current VS2022 builds to be consistent with the VC6.